### ISSUE TITLE
[Bug]: Noticed inconsistency to add title, rel, ARIA Attributes to Social Links& Needs ARIA-attributes to Form Section

### ISSUE THREAD

**Issue by NarendraKoya999** [Opentek-Org/opentek#567]
### Description

- ` In Main Page like index.html File have target, title, aria-label, rel attributes Added to Social Links`
- `So, I want to add those above Missing attributes to remaining 10 HTML Files occurred in pages Folder due to purpose of Maintaining Consistanacy.`
- `Also, I want to add Require ARIA Attributes to Form Section. This is missing in Main index.html File `

### Additional information

- The `aria-label` can be used in cases where text that could label the element is not visible. If there is visible text that labels an element, use aria-labelledby instead. The purpose of aria-label is the same as aria-labelledby . Both provide an accessible name for an element.
- The `rel attribute` specifies the relationship between the current document and the linked document. Only used if the href attribute is present.
- The `title attribute` gives users an important indication as to where they will be directed to when they click on a link. If the user hovers with the mouse over a link reference, the title text, which was defined in the attribute, will be displayed.

### What browser are you seeing the problem on?

Chrome

### Checklist

- [X] I have checked the [existing issues](https://github.com/Opentek-Org/opentek/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/Opentek-Org/opentek/blob/main/CONTRIBUTING.md)
- [X] I want to work on this issue. (optional)
- [X] I am a GSSoC'23 contributor

**Comment by NarendraKoya999** [comment]
Hi @AnirudhPanda . Please Review on this PR. Also, I have one Doubt . Is any chances to add Other Job Portal Websites.
If Yes, I am sharing that Resources. These Resources are Most underrated.

**Comment by AnirudhPanda** [comment]
Okay
